config.yaml.example

```diff
@@ -40,3 +40,9 @@ aws:
       ports:
         - 22
         - 2222
+gsuite:
+  domain: 'example.com'
+  user_is_inactive:
```

Comment:
I wonder if we should abstract this into a cross-provider config value in the future (since we'd probably have the same policy across providers).